### PR TITLE
Improve success message

### PR DESCRIPTION
**Pull request by codemist**

### CODE REVIEW

**Comment by maxxcrawford** [comment]
@groovecoder We had a force-push issue (my fault) and the previous PR (#915) was closed. 😅

File: privaterelay/templates/home.html [hunk 1]
```diff
@@ -7,23 +7,25 @@
...
-		</div>
-		<div class="hero-right flx flx-col">
-      <div class="hero-1 hero-slide"></div>
-      <div class="hero-2 hero-slide"></div>
-      <div class="hero-3 hero-slide"></div>
-      <div class="hero-4 hero-slide"></div>
-      <div class="hero-5 hero-slide"></div>
-		</div>
-	</div>
+<!--{% include "includes/messages.html" %}-->
```
Comment (maxxcrawford) [inline privaterelay/templates/home.html:10]: Can you remove this commented line of code?

File: static/scss/partials/main.scss [hunk 2]
```diff
@@ -31,6 +31,7 @@
   --violet3: #9059ff;
   --violet4: #7542e5;
   --warningRed: #d70022;
+  --successGreen: #00D053;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:34]: Let's migrate to Nebula's green. (Confirm with Eduardo)

Additionally – let's start a new variable calling it the correct color. `$color-green-50`. 

Screen Shot 2021-07-26 at 12 48 30 PM


image

File: static/scss/partials/main.scss [hunk 3]
```diff
@@ -2367,12 +2368,20 @@ input.input-has-error {
   }
 }
 
+.messages {
+  width:100%;
+  padding:1.5rem 16px 0 16px;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:2373]: Let's not mix `rem` and `px`. 

Can you use `rem` for all of it?

File: static/scss/partials/main.scss [hunk 4]
```diff
@@ -2367,12 +2368,20 @@ input.input-has-error {
...
+}
+
 .messages > div {
   background-color: var(--grey50);
 }
 
 .messages .success {
-  background-color: var(--blue7);
+  background-color: var(--successGreen);
+  border-radius:8px;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:2383]: Use the [Protocol spacing token](https://protocol.mozilla.org/fundamentals/tokens.html) here: `$spacing-sm`

File: static/scss/partials/main.scss [hunk 5]
```diff
@@ -2406,7 +2413,7 @@ ul.messages {
   margin: auto;
   outline: none;
   border: none;
-  background: rgba(0, 0, 0, 0.5);
+  background: rgba(255, 255, 255, 0);
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:2416]: Use `$color-white` here instead of RGBA.

File: static/scss/partials/main.scss [hunk 6]
```diff
@@ -2415,6 +2422,9 @@ ul.messages {
 
 .home-messages span {
   color: $color-white;
+  text-align: center;
+  width:100%;
+  font-weight:600;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:2427]: nit: spacing

```suggestion
  width: 100%;
  font-weight: 600;
```

File: static/scss/partials/main.scss (hunk 4 above)
Comment (maxxcrawford) [inline static/scss/partials/main.scss:2383]: nit: spacing

```suggestion
  border-radius: 8px;
```

File: static/scss/partials/main.scss (hunk 3 above)
Comment (maxxcrawford) [inline static/scss/partials/main.scss:2373]: nit: spacing
```suggestion
  width: 100%;
  padding: 1.5rem 16px 0 16px;
```

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
Looks good! Mostly nits and Protocol refinements.

File: static/scss/partials/main.scss (hunk 2 above)
Comment (maxxcrawford) [reply static/scss/partials/main.scss:34]: Actually – one second thought. Leave this variable (`successGreen`) as is. I'm going to add in all the colors and we can migrate this later. 

If you wanna update the color of the variabel to `#3FE1B0`, go for it.

Comment (maxxcrawford) [reply static/scss/partials/main.scss:34]: (Issue filed as https://github.com/mozilla/fx-private-relay/issues/961 and assigned to me)

File: static/scss/partials/main.scss (hunk 3 above)
Comment (codemist) [reply static/scss/partials/main.scss:2373]: For this, should I be using protocol?

**Comment by codemist** [comment]
Preview:
image

File: static/scss/partials/main.scss [hunk 7]
```diff
@@ -2392,10 +2401,8 @@ ul.messages {
   padding: 1rem;
   align-items: center;
   display: flex;
-  
-  border-radius: 8px;
-  max-width: 960px;
   margin: 0 auto;
+  text-align:center;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:2405]: nit: whitespace 
```suggestion
  text-align: center;
```

File: static/scss/partials/main.scss [hunk 8]
```diff
@@ -2367,12 +2368,20 @@ input.input-has-error {
   }
 }
 
+.messages {
+  width: 100%;
+  padding: $spacing-lg $spacing-md 0 $spacing-md;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:2373]: 🙌

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
Found one last `whitespace` issue. 🙈

**Review by maxxcrawford — APPROVED** [review]
Let's ship it! 🍃

**Pull request merged**